eng/tools/azure-sdk-tools/azpysdk/samples.py

```diff
@@ -0,0 +1,342 @@
+        "connection_to_custom_endpoint_address.py",
+        "proxy.py",
+        "connection_to_custom_endpoint_address_async.py",
+        "iot_hub_connection_string_receive_async.py",
+        "proxy_async.py",
+        "send_stream.py",
+        "send_stream_async.py",
+        "send_buffered_mode.py",
+        "send_buffered_mode_async.py",
+        "send_and_receive_amqp_annotated_message.py",
+        "send_and_receive_amqp_annotated_message_async.py",
+    ],
+    "azure-eventhub-checkpointstoretable": ["receive_events_using_checkpoint_store.py"],
+    "azure-servicebus": [
+        "connection_to_custom_endpoint_address.py",
+        "mgmt_queue.py",
+        "mgmt_rule.py",
+        "mgmt_subscription.py",
+        "mgmt_topic.py",
+        "proxy.py",
+        "receive_deferred_message_queue.py",
+        "connection_to_custom_endpoint_address_async.py",
+        "mgmt_queue_async.py",
+        "mgmt_rule_async.py",
+        "mgmt_subscription_async.py",
+        "mgmt_topic_async.py",
+        "proxy_async.py",
+        "receive_deferred_message_queue_async.py",
+    ],
+    "azure-communication-chat": [
+        "chat_client_sample_async.py",
+        "chat_client_sample.py",
+        "chat_thread_client_sample_async.py",
+        "chat_thread_client_sample.py",
+    ],
+    "azure-communication-phonenumbers": [
+        "purchase_phone_number_sample_async.py",
+        "purchase_phone_number_sample.py",
+        "release_phone_number_sample_async.py",
+        "release_phone_number_sample.py",
+    ],
+    "azure-ai-translation-document": [
+        "sample_list_document_statuses_with_filters_async.py",
+        "sample_list_translations_with_filters_async.py",
+        "sample_list_document_statuses_with_filters.py",
+        "sample_list_translations_with_filters.py",
+        "sample_translation_with_custom_model.py",
+        "sample_translation_with_custom_model_async.py",
+        "sample_begin_translation_with_filters.py",
+        "sample_begin_translation_with_filters_async.py",
+    ],
+    "azure-ai-language-questionanswering": ["sample_export_import_project.py", "sample_export_import_project_async.py"],
+    "azure-ai-textanalytics": [
+        "sample_analyze_healthcare_entities_with_cancellation.py",
+        "sample_analyze_healthcare_entities_with_cancellation_async.py",
+        "sample_abstract_summary.py",
+        "sample_abstract_summary_async.py",
+    ],
+    "azure-storage-blob": [
+        "blob_samples_proxy_configuration.py",
+        "blob_samples_container_access_policy.py",
+        "blob_samples_container_access_policy_async.py",
+        "blob_samples_client_side_encryption_keyvault.py",
+    ],
+}
+
+
+def run_check_call_with_timeout(
+    command_array,
+    working_directory,
+    timeout,
+    pass_if_timeout,
+    acceptable_return_codes=[],
+    always_exit=False,
+):
+    """This is copied from common_tasks.py with some additions.
+    Don't want to break anyone that's using the original code.
```

Comment:
I also feel like if we aren't going to update the implementation when copying, we should just call this new location from the existing code. This may not be possible but take a look at that?